test/IISTestSite/Startup.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+
+namespace IISTestSite
```

Comment:
ServerVariableTestSite? Or do you plan on expanding this?

Reply:
From what I can tell, most of the tests in this project don't need to deploy a new instance of the IISTestSite. I added it this way and opened #565 to simplify the functional testing to share a common instance of IISTestSite.

Reply:
We can overload the single published site with all of our tests to reduce unnecessary deployments. We can also modify the deployer to use different configurations with an already published application.